3rd-party-licenses.txt

```diff
@@ -1,8 +1,40 @@
 Third Party Licenses
 ====================
 
-This repository contains multiple copies of the project Lab Project FreeRTOS POSIX released under a MIT License: https://github.com/FreeRTOS/Lab-Project-FreeRTOS-POSIX.
-The Lab Project FreeRTOS POSIX MIT license is:
+freertos_apps includes material from the projects listed below (Third Party
+IP). The original copyright notice and the license under which we received
+such Third Party IP, are set forth below.
+
+
+--------------------------------------------------------------------------
+Overview
+--------------------------------------------------------------------------
+
+Multiple folders contain copies of the project Lab Project FreeRTOS POSIX,
+licensed under the MIT license.
+
+Name:      Lab Project FreeRTOS POSIX
+Version:   <<fill version number of commit id>>
```

Comment:
@pablogs9, can you please provide the version number?

Reply:
Done